### PR TITLE
collapsible fix

### PR DESCRIPTION
**Pull request by galloween**
- this fixes issues that happened if collapsible panel started expanded (animation problems)
- also streamlines some css (not-divided option) added by @avgdorom recently

### CODE REVIEW

File: projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section.component.html [hunk 1]
```diff
@@ -30,19 +33,19 @@
...
+         [ngClass]="{'bcp-header-content-clickable': collapsible && !options.headerTranscludeStopPropagation}"
          (click.outside-zone)="options.headerTranscludeStopPropagation && $event.stopPropagation()">
       <ng-content select="[header]"></ng-content>
     </div>
 
   </header>
 
   <div class="bcp-panel"
-       *ngIf="!collapsible || (collapsible && contentLoaded)"
+       *ngIf="!collapsible || expanded || (collapsible && contentLoaded)"
```
Comment (avgdorom) [inline projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section.component.html]: Can be reduced to `!collapsible || expanded || contentLoaded`, cause if you get to `contentLoaded` you already know that `collapsible === true` 😄

File: projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section.component.html [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
 
   <header class="
          bcp-header"
           [attr.id]="panelID+'-header'"
           [attr.aria-controls]="(collapsible && panelID) || null"
           [attr.aria-expanded]="(collapsible && expanded) || null"
           [ngClass]="{
-          'b-icon-chevron-right': collapsible && !expanded,
-          'b-icon-chevron-down': collapsible && expanded,
-          'no-shadow': collapsible && expanded && !divided
```
Comment (avgdorom) [inline projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section.component.html]: Are you sure about removing this? I found out that leaving the shadow creates a pseudo-divider too, apart from the border of the collapsible part...

Comment (avgdorom) [reply projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section.component.html]: Oh!!!! Just saw what you did there 😉

File: projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section.component.html (hunk 1 above)
Comment (galloween) [reply projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section.component.html]: 👍

File: projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section.component.html (hunk 2 above)
Comment (galloween) [reply projects/ui-framework/src/lib/layout/collapsible-section/collapsible-section.component.html]: 👻

**Pull request merged**